pkg/storage/client_raft_test.go

```diff
@@ -301,7 +301,15 @@ func TestReplicateRange(t *testing.T) {
 // persisted to disk and restored when a node is stopped and restarted.
 func TestRestoreReplicas(t *testing.T) {
 	defer leaktest.AfterTest(t)()
-	mtc := startMultiTestContext(t, 2)
+
+	sc := storage.TestStoreConfig()
+	// Disable periodic gossip activities. The periodic gossiping of the first
+	// range can cause spurious lease transfers which cause this test to fail.
+	sc.TestingKnobs.DisablePeriodicGossips = true
+	mtc := &multiTestContext{
```

Comment:
`mtc.Start` takes a pointer receiver. I prefer the explicitness. Also, there are some global methods with take `*multiTestContext`. We're a bit inconsistent here, so I've gone ahead and changed all of the `mtc` variables to pointers.